src/commands/show-lists.js

```diff
@@ -2,70 +2,105 @@
 
 fs = require("fs");
 
+var _ = require("underscore");
+
 var __ = function(program, output, logger, config, trello, translator, trelloApiCommands) {
 
     var trelloApiCommand = {};
 
-    trelloApiCommand.makeTrelloApiCall = function (options, onComplete) {
+    trelloApiCommand.makeTrelloApiCall = function(options, onComplete) {
         logger.info("Showing lists belonging to the specified board");
 
+        var boardIds = [];
+
         // Grab our boards etc
-        try {
-            var boardId = translator.getBoardIdByName(options.board);
-        } catch (err) {
-            if (err.message == "Unknown Board") {
-                logger.warning("Unknown board.  Perhaps you have a typo?");
-
-                output.normal("\nYou have the following open boards:\n");
-                trelloApiCommands["show-boards"].makeTrelloApiCall({ includeClosed : false, hideIds : true}, null);
-                return;
+        if (options.board) {
+            try {
+                boardIds.push(translator.getBoardIdByName(options.board));
+            } catch (err) {
+                if (err.message == "Unknown Board") {
+                    logger.warning("Unknown board.  Perhaps you have a typo?");
+
+                    output.normal("\nYou have the following open boards:\n");
+                    trelloApiCommands["show-boards"].makeTrelloApiCall({
+                        includeClosed: false,
+                        hideIds: true
+                    }, null);
+                    return;
+                }
             }
+        } else {
+            _.each(translator.cache.translations.boards, function(oneBoard, boardId) {
+                if (boardId != "undefined") {
+                    boardIds.push(boardId);
+                }
+            });
         }
-        var boardId = translator.getBoardIdByName(options.board);
-
-        trello.get("/1/boards/" + boardId + "/lists",
-						{}, function(err, data) {
-            if (err) throw err;
-
-						if ((!!data) && data.length > 0) {
-							if (options.showListName) {
-								output.normal(translator.getList(data[0].id).underline);
-							} else {
-								output.normal(translator.getBoard(data[0].idBoard).underline);
-							}
-							for (var i = 0 ; i < data.length ; i++) {
-									var formattedCardName = data[i].name.replace(/\n/g, "");
-									output.normal("* " + formattedCardName);
-							}
-						}
 
+        boardIds.forEach(function(boardId) {
+            var listOfLists = [];
+            _.each(translator.cache.translations.lists, function(oneList, listId) {
+                if (listId != "undefined" && (oneList["board"] == boardId)) {
+                    // oneList: [ boardId, listName ]
+                    listOfLists.splice(
+                        _.sortedIndex(listOfLists, {
+                            id: listId,
+                            name: oneList["name"]
```

Comment:
I'm not following what you mean, @mheap?  Would you be willing to further explain?  (I also just woke up, my morning brain may have something to do with that... XD)